BoxContentSDK/BoxContentSDK/Operations/BOXAPIAppUsersAuthOperation.m

```diff
@@ -42,18 +42,19 @@ - (void)executeOperation
 {
     if (![self isCancelled]) {
         @synchronized(self.session) {
-            __weak BOXAPIAppUsersAuthOperation *weakSelf = self;
             [self.session.queueManager.delegate fetchAccessTokenWithCompletion:^(NSString *accessToken, NSDate *accessTokenExpiration, NSError *error) {
                 if (error == nil && accessToken && accessTokenExpiration) {
-                    weakSelf.accessToken = accessToken;
-                    weakSelf.accessTokenExpiration = accessTokenExpiration;
+                    self.accessToken = accessToken;
+                    self.accessTokenExpiration = accessTokenExpiration;
                 } else if (!error) {
-                    self.error = [[NSError alloc] initWithDomain:@"accessToken and accessTokenExpiration should be non-nil" code:BOXContentSDKAppUserErrorAccessTokenInvalid | BOXContentSDKAppUserErrorAccessTokenExpirationInvalid userInfo:nil];
+                    self.error = [[NSError alloc] initWithDomain:@"accessToken and accessTokenExpiration should be non-nil"
+                                                            code:BOXContentSDKAppUserErrorAccessTokenInvalid
```

Comment:
Note, I changed the code here from a logical OR of two codes to a single error code, as it is meant to be used.